templates/base/eslint.config.mjs

```diff
@@ -17,5 +17,16 @@ export default [
   pluginJs.configs.recommended,
   ...tseslint.configs.recommended,
   pluginReact.configs.flat.recommended,
-  eslintPluginPrettierRecommended,  
+  eslintPluginPrettierRecommended,
+  {
+    name: 'Custom Rules',
+    rules: {
+      'no-console': 2,
+      'react/react-in-jsx-scope': 'off',
+      'react/no-array-index-key': 2,
+      'react-hooks/rules-of-hooks': 2, // Checks rules of Hooks
```

Comment:
Comment seems redundant to the name of the rule. I am all for comments, but this seems a bit lost and random